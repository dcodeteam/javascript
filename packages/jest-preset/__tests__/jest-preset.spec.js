"use strict";

// eslint-disable-next-line import/no-extraneous-dependencies
const { initJestPresetTest } = require("test-tools");

initJestPresetTest("jest-preset", require.resolve("../jest-preset"));
