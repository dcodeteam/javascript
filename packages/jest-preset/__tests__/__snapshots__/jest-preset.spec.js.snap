// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-preset should match snapshot 1`] = `
Object {
  "coveragePathIgnorePatterns": Array [
    "/__docs__/",
    "/__mocks__/",
    "/__tests__/",
    "/__testutils__/",
  ],
  "coverageThreshold": Object {
    "global": Object {
      "branches": 50,
      "functions": 50,
      "lines": 50,
      "statements": 50,
    },
  },
  "moduleFileExtensions": Array [
    "ts",
    "tsx",
    "js",
    "json",
    "jsx",
    "node",
  ],
  "setupTestFrameworkScriptFile": "<rootDir>/src/__testutils__/setupTests",
  "testEnvironment": "node",
  "testMatch": Array [
    "**/__tests__/**/*.spec.ts?(x)",
  ],
  "testRunner": "jest-circus/runner",
  "transform": Object {
    "^.+\\\\.(ts|tsx)$": "babel-jest",
  },
}
`;
